### ISSUE TITLE
Option to hide end cards

### ISSUE THREAD

**Issue by ANewDawn** [webosbrew/youtube-webos#126]
An option to hide end cards that show up at the end of videos as they block the view and you miss what is seen at the end.

![downloadfile](https://github.com/webosbrew/youtube-webos/assets/51456745/5da3896f-aaad-42b3-b9e7-2e205b9d28b0)

**Comment by ANewDawn** [comment]
Hi @NicholasBly

Any chance could get look removing these end cards if it's possible.

Cheers

**Comment by NicholasBly** [comment]
> Hi [@NicholasBly](https://github.com/NicholasBly)
> 
> Any chance could get look removing these end cards if it's possible.
> 
> Cheers

Sure I'll look into it

**Comment by NicholasBly** [comment]
https://github.com/NicholasBly/youtube-webos/actions/runs/16107170247

Available for testing! Currently removes the entire endcard, however some people might want to prefer removing the thumbnail only so they can still see the endcard without blocking the video, but for now just open the menu and uncheck that box and it'll immediately show endcards again.

**Comment by ANewDawn** [comment]
> https://github.com/NicholasBly/youtube-webos/actions/runs/16107170247
> 
> Available for testing! Currently removes the entire endcard, however some people might want to prefer removing the thumbnail only so they can still see the endcard without blocking the video, but for now just open the menu and uncheck that box and it'll immediately show endcards again.

Works perfect. Much appreciate your quick work on this. Cheers 👍

**Comment by ANewDawn** [comment]
@NicholasBly 

There seems bug with version above. When typing in search bar auto complete words aren't showing under it. Could you check on your end.

Also any chance of removing the sign in nag under the seekbar as it blocks queued videos being seen without having to press down arrow see them. See screenshot

Many thanks 

![Image](https://github.com/user-attachments/assets/1eaefdd0-3013-4987-a49f-241f15c7b95e)

**Comment by NicholasBly** [comment]
> [@NicholasBly](https://github.com/NicholasBly)
> 
> There seems bug with version above. When typing in search bar auto complete words aren't showing under it. Could you check on your end.

Good catch. I went back to 0.3.9 and that causes the issue too, so this bug has been there before I contributed to this project. Each keystroke throws an error - TypeError: Cannot read properties of undefined. I'll try to have a fix soon.

Edit: Looks to be caused by adblock being enabled. If you uncheck the adblock feature, search works. Working on a fix.

**Comment by NicholasBly** [comment]
@ANewDawn Search suggestion fix available here - https://github.com/NicholasBly/youtube-webos/actions/runs/16137738488

I'll look at your sign in box tomorrow 👍

**Comment by ANewDawn** [comment]
> [@ANewDawn](https://github.com/ANewDawn) Search suggestion fix available here - https://github.com/NicholasBly/youtube-webos/actions/runs/16137738488
> 
> I'll look at your sign in box tomorrow 👍

Search working. Cheers again 😀

**Comment by NicholasBly** [comment]
@ANewDawn Hey sorry I wasn't successful adjusting the guest mode button, I got as far as hiding the button but can't figure out how to move the up next section up.

**Comment by ANewDawn** [comment]
> [@ANewDawn](https://github.com/ANewDawn) Hey sorry I wasn't successful adjusting the guest mode button, I got as far as hiding the button but can't figure out how to move the up next section up.

Ah no worries. It's only minor nag. Thanks for help anyways 🙂

**Comment by ANewDawn** [comment]
> [@ANewDawn](https://github.com/ANewDawn) Hey sorry I wasn't successful adjusting the guest mode button, I got as far as hiding the button but can't figure out how to move the up next section up.

Hi Nicholas

I wouldn't mind testing with the button Hidden without needing to move section up. Hiding it should be good enough as that button blocks most of screen.

Thanks again
